pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
Seems like a more natural fit inside of `schema1ToImage` to build a complete `Image` object.  The additional fanout is less than beautiful, but it would keep the API looking more consistent.  `schema1ToImage` would return a complete and valid look.